lighthouse-cli/index.js

```diff
@@ -77,6 +78,13 @@ if (cli.flags.listAllAudits) {
   process.exit(0);
 }
 
+if (cli.flags.listTraceCategories) {
+  const categories = lighthouse.traceCategories;
+
+  log.info('All trace categories required by lighthouse:', categories.join(', '));
```

Comment:
We should then consider dropping the log require, or make it consistent with https://github.com/mgerakis/lighthouse/blob/46d3431ecf5a60b37aca98233b85ecc77ac5f0a5/lighthouse-cli/index.js#L77